tooling/Microsoft.VisualStudio.BlazorExtension/Microsoft.VisualStudio.BlazorExtension.csproj

```diff
@@ -103,12 +109,24 @@
       this problem, we're only using the P2P references to force build ordering. Then we include the build
       output of those projects as content.
     -->
+    <Content Include="Templates.pkgdef" />
+    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Blazor.Templates\Microsoft.AspNetCore.Blazor.Templates.csproj">
+      <Project>{edd21533-c6e6-4f85-be4f-10e06756e24c}</Project>
+      <Name>Microsoft.AspNetCore.Blazor.Templates</Name>
+      <Private>False</Private>
```

Comment:
If you're still having trouble getting this to build, you could do something like this in the templates project

<PropertyGroup> 
  <TargetFrameworks Condition="'$(OS)'=='Windows_NT'">net46;netstandard2.0</TargetFrameworks>
  <TargetFrameworks Condition="'$(OS)'!='Windows_NT'">netstandard2.0</TargetFrameworks>
</PropertyGroup

Reply:
Thanks Ryan! No, netstandard2.0 fix it. We are  good to go if everyone is good.